api.yaml

```diff
@@ -302,11 +298,9 @@ paths:
           async: false
           request: >
             {
-              #! if .Request.Query.days !#
-              "days": #!index .Request.Query.days 0!#
-              #! end !#
+              "name": #!index .Request.Query.name 0!#
             }
-          success: '{"status": 200, "bodyPassthrough": true}'
+          success: '{"status": 204, "bodyPassthrough": true}'
```

Comment:
Nit: Given that this returns a 204, you should probably drop the `bodyPassthrough`